examples/posturemanagementv1/CreateCollector.go

```diff
@@ -0,0 +1,29 @@
+package examples
```

Comment:
Examples will be auto generated on passing --genExamples and will appear inside the posturemanagementv1 folder. For example https://github.com/IBM/scc-go-sdk/blob/main/findingsv1/findings_v1_examples_test.go

Reply:
examples have to be generated only, we don't need manual examples... please delete this after you've generated examples

Reply:
accepted and updated